File: Javascript/Javascript/03 - Funciones/funciones.js
function saludar(nombre){
    //console.log("Hola desde la función saludar");
   // console.log(`Hola ${nombre} desde la función saludar`);
   return `Hola ${nombre} desde la función saludar`

}

const s = saludar("Andres");
const r = saludar("Juan");

//console.log(s);
//console.log(r);



function sumar(a=null,b=null){
    if(a==null || b==null){
        console.log("Error debes colocar dos numeros en la función");
        return;
    }
    return a+b;
}

const c = sumar(4,9);
console.log(c);

File: Java/CursoJava/src/Funciones.java

public class Funciones {
    public static void main(String[] args) {
        var suma = sumar(5, 6);
        System.out.println("La suma es: "+suma);
        
        var nombre = "Juan";
        var edad = 25;
        saludar(nombre, edad);
        //saludar("Juan",23); otra forma
        
    }
    
    static int sumar(int a, int b){
        return a+b;
    }
    
    static void saludar(String nombre, int edad){
        System.out.printf("Hola %s tu edad es %d\n",nombre, edad);
    }
}
